src/controllers/invoices.js

```diff
@@ -98,7 +98,7 @@ export default function () {
           ${(shippingLine
             ? `
             ${i18n(i19zipCode)}: ${formatCEP(shippingLine.to.zip)}<br>
-            <address>${getLineAddress(shippingLine.to)}</address>`
+            <address>${getLineAddress(shippingLine.to)} ${(shippingLine.to.near_to ? `<br>Referência: ${shippingLine.to.near_to}` : '')}</address>`
```

Comment:
```suggestion
            <address>${getLineAddress(shippingLine.to)} ${(shippingLine.to.near_to ? `<br>Ref.: ${shippingLine.to.near_to}` : '')}</address>`
```

Melhor, ou não dá pra entender resumido assim? Bom que funciona pra inglês também 😬

Reply:
Acredito que sim, bora testar